docs/faq/index.md

```diff
@@ -28,7 +28,7 @@ If you don't see your question here, please reach out to us on Slack or at
     We're constantly adding new data types and annotation types — if you don't see what you're looking for, reach out
     to us and we'll happily extend our system to support your use case.
 
-??? faq "Do I have to upload my datasets to Kolena?"
+??? faq "Do I have to upload my data files to Kolena?"
```

Comment:
I would consider updating to "images, documents, or files"

Reply:
Done.